app/main.py

```diff
@@ -14,4 +14,5 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    return [let for let in phrase.lower()].count(letter.lower())
```

Comment:
Why is a loop needed here?
```suggestion
    return phrase.lower().count(letter.lower())
```

Reply:
I fixed it. Can you look?